app/models/event/course_record.rb

```diff
@@ -192,6 +196,14 @@ def set_defaults
     true # ensure callback chain continues
   end
 
+  def duration_in_days?
+    return !duration_in_hours?
```

Comment:
dieses `return` würde ich wegnehme. Es ist zwar so eine Art guard-clause, aber dann eben doch der ganze Methodenbody. 

```suggestion
    !duration_in_hours?
```